FND/GoalsAndObjectives/Objectives.rdf

```diff
@@ -78,11 +78,30 @@ THE SOFTWARE IS PROVIDED &apos;AS IS&apos;, WITHOUT WARRANTY OF ANY KIND, EXPRES
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20230101/GoalsAndObjectives/Objectives.rdf version of this ontology was modified to use the Commons Ontology Library (Commons) rather than the OMG&apos;s Languages, Countries and Codes (LCC), eliminating redundancies in FIBO as appropriate.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20230301/GoalsAndObjectives/Objectives.rdf version of this ontology was modified to add the concept of a project and related attributes (LOAN-169).</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20241101/GoalsAndObjectives/Objectives.rdf version of this ontology was modified to add the concept of a method (SEC-200) and add nonprofit investment objectives (SEC-208).</skos:changeNote>
+		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20250901/GoalsAndObjectives/Objectives.rdf version of this ontology was modified to add a parent concept for goal and objective, and a parent concept for strategy and method, and adjust definitions accordingly (FND-402).</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 		<cmns-av:copyright>Copyright (c) 2013-2025 EDM Council, Inc.</cmns-av:copyright>
 		<cmns-av:copyright>Copyright (c) 2013-2025 Object Management Group, Inc.</cmns-av:copyright>
 	</owl:Ontology>
 	
+	<owl:Class rdf:about="&fibo-fnd-gao-obj;Aim">
+		<rdfs:label>aim</rdfs:label>
+		<skos:definition>intended end state characterized by desired outcomes toward which actions are directed, regardless of scope or measurability</skos:definition>
+		<cmns-av:explanatoryNote>Business aims are sometimes considered broad, long-term goals, and in such cases, the term &apos;aim&apos; is used interchangeably with goal. Here, however we differentiate between qualitative goals and quantitative objectives, both of which are kinds of aims, with the critical differences including measurability and time frame. Goals tend to have a much longer trajectory, provide the basis for determining objectives, and are often aligned with an organization&apos;s mission, whereas objectives are short term and measurable.</cmns-av:explanatoryNote>
```

Comment:
this note relates to subclasses, not Aim, so not sure it belongs here

Reply:
> this note relates to subclasses, not Aim, so not sure it belongs here

It really provides some additional differentiation between the terms, which are often overloaded. I'd rather keep it as I think it's better here than on either goal or objective.